### PR TITLE
Add TCS icon

### LINKED ISSUE #5323: TCS
**Brand Name:** TCS (Tata Consultancy Services)
**Website:** https://www.tcs.com/
**Alexa rank:** 3,792
  

**Official resources for icon and color:** https://pbs.twimg.com/profile_images/1319549897515806722/rSiFW4qo_400x400.jpg

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
tcs

**Issue:** closes #5323 

**Similarweb rank:** 18,811. See https://www.similarweb.com/website/tcs.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website footer https://www.tcs.com/. Header is PNG.
COlor: `#ee3984` Taken from one of the logo colors. I am not sure. Maybe `#000000` would be more suitable.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11862,6 +11862,16 @@
...
         },
+        {
+            "title": "TCS",
+            "hex": "EE3984",
+            "source": "https://www.tcs.com",
+            "aliases": {
+                "aka": [
+                    "Tata Consultancy Services"
+                ]
+            }
```
Comment (adamrusted) [inline _data/simple-icons.json:11873]:
```suggestion
            "title": "Tata Consultancy Services",
            "slug": "tcs",
            "hex": "EE3984",
            "source": "https://www.tcs.com",
            "aliases": {
                "aka": [
                    "TCS"
                ]
            }
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
I would, in this instance, suggest we display the full title - but use the shorter slug. Purely because the 'aka' tag isn't used on the website at the minute. Otherwise, icon is spot on!

**Comment by PratyushJayachandran** [comment]
> I would, in this instance, suggest we display the full title - but use the shorter slug. Purely because the 'aka' tag isn't used on the website at the minute. Otherwise, icon is spot on!

Applied your suggestion

**Comment by PratyushJayachandran** [comment]
> I would, in this instance, suggest we display the full title - but use the shorter slug. Purely because the 'aka' tag isn't used on the website at the minute. Otherwise, icon is spot on!

Applied your suggestion

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PratyushJayachandran 🎉

**Pull request merged**